### PR TITLE
#47:Parameter View - ordering of parameters

### PR DESCRIPTION
**Pull request by MarkWPeter**
Feature implemented, use of order_priority defines order within category, as described in issue #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,8 @@ class ParameterDefinitions(TapirParameterDefinitionImporter):
             datatype=TapirParameterDatatype.STRING,
             initial_value="WirGarten Lüneburg eG",
             description="Der Name des WirGarten Standorts. Beispiel: 'WirGarten Lüneburg eG'",
-            category=ParameterCategory.SITE,
+            category=ParameterCa~~~~tegory.SITE,
```
Comment (mk2301) [inline README.md:152]: wtf?

File: docker-compose.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ services:
   nginx-proxy:
     image: jwilder/nginx-proxy
     volumes:
-      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - /run/user/1000/docker.sock:/tmp/docker.sock:ro
```
Comment (mk2301) [inline docker-compose.yml:35]: pls don't commit docker-compose.yml

File: tapir/celery.py [hunk 3]
```diff
@@ -8,7 +8,6 @@
 app = Celery("tapir")
 
 # Using a string here means the worker doesn't have to serialize
```
Comment (mk2301) [inline tapir/celery.py:10]: deleted comment by accident?

**Comment by MarkWPeter** [comment]
Sorry, had a hickup with git stash which compromised my last commit. It should be fixed now.

**Comment by mk2301** [comment]
I removed the poetry.lock from the commit, not sure if it is safe to update right now. Will be handled and updated in #54

**Review by mk2301 — APPROVED** [review]

**Pull request merged**